packages/react-native-payments/lib/js/PaymentRequest.js

```diff
@@ -403,10 +403,10 @@ export default class PaymentRequest {
     return this._acceptPromiseRejecter(new GatewayError(details.error));
   }
 
-  _closePaymentRequest() {
+  _closePaymentRequest(reject = true) {
```

Comment:
this gets called by a USER_DISMISS_EVENT emit, are you sure there are no args passed when that's emitted?

Reply:
looks like it's fine: https://github.com/ExodusMovement/react-native-payments/blob/9ec6218ed7623f1737820d3577eda7090384fb20/packages/react-native-payments/lib/ios/ReactNativePayments.m#L102